ci/README.md

```diff
@@ -18,6 +18,40 @@ The goal is to eventually have all GitHub specific code in `ci/github-script` an
 A lot of code has already been migrated, but some Bash code still remains.
 New CI features need to be introduced in JavaScript, not Bash.
 
+## Nixpkgs merge bot
+
+The Nixpkgs merge bot empowers package maintainers by enabling them to merge PRs related to their own packages.
+It serves as a bridge for maintainers to quickly respond to user feedback, facilitating a more self-reliant approach.
+Especially when considering the scale of [3’976 maintainers][maintainers] compared to just [235 committers][committers], this bot is a game-changer.
+
+Following [RFC 172] the merge bot was originally implemented as a [python webapp](https://github.com/NixOS/nixpkgs-merge-bot), which has now been integrated into [`ci/github-script/bot.js`](./github-script/bot.js) and [`ci/github-script/merge.js`](./github-script/merge.js).
+
+### Using the merge bot
+
+To merge a PR, maintainers can simply comment:
+```gfm
+@NixOS/nixpkgs-merge-bot merge
+```
+
+The next time the bot runs it will verify the below constraints, then (if satisfied) merge the PR.
+
+The merge bot will reference #306934 on PRs it merges successfully, while unsuccessful attempts will reference #305350.
+These issues effectively list PRs the merge bot has interacted with.
```

Comment:
There's also the rare case of #371492 for failed merges, where "failed" is because of some errors, not because of not satisfying the checklist.